.gitignore

```diff
@@ -1,2 +1,6 @@
 # Golang dynamic libs/pluging
-*.so
\ No newline at end of file
+*.so
+
+# Local dev files
+/go.work
```

Comment:
can this just be go.work? i.e. no slash

Reply:
I would agree usually, but I don't think go.work _can_ live at any other location? 🤔